edx_solutions_api_integration/imports/views.py

```diff
@@ -207,9 +207,10 @@ def _enroll_user(self, data, errors, response={}):
         cohort = get_cohort_by_name(course_key, CourseUserGroup.default_cohort_name)
 
         try:
-            CohortMembership.objects.create(course_user_group=cohort, user=user)
-        except IntegrityError:
-            # This situation can occur if user is already present in the course or if his enrollment was removed
+            add_user_to_cohort(cohort, user.username)
+        except ValueError:
+            # This situation can occur if user is already present in the cohort probably because they were un-enrolled
+            # and re-enrolled or if his enrollment was removed
```

Comment:
The message added here should be telling why would ValueError occur, rather than the reason you changed the try block code.